includes/Admin/Admin_AJAX.php

```diff
@@ -59,7 +157,7 @@ public function get_checks_to_run() {
 			wp_send_json_error( $valid_nonce, 403 );
 		}
 
-		$checks = filter_input( INPUT_POST, 'checks', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+		$checks = wp_parse_list( filter_input( INPUT_POST, 'checks', FILTER_SANITIZE_STRING ) );
```

Comment:
See above, I don't think we should change this.

Reply:
Updated